alacritty{-graphics}: 0.15.1 -> 0.16.0

ddogfoodd opened the pull request:
Changelog: https://github.com/alacritty/alacritty/releases/tag/v0.16.0
Diff: https://github.com/alacritty/alacritty/compare/v0.15.1...v0.16.0

Changelog: https://github.com/ayosec/alacritty/releases/tag/v0.16.0-graphics
Diff: https://github.com/ayosec/alacritty/compare/v0.15.1-graphics...v0.16.0-graphics

## Things done
- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

afh reviewed (changes requested):
I suggest to convert this to a draft until the [graphics fork](https://github.com/ayosec/alacritty) has been updated.

If folks not using the graphics fork are keen on getting this in I suggest changing the version based on the `withGraphics` attribute to ensure that both variants continue to work properly.

afh commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 453142`
Commit: `66b50584757d9dbd82039205d0401879039c1130`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>alacritty</li>
    <li>alacritty-graphics</li>
    <li>alacritty-graphics.terminfo</li>
    <li>alacritty.terminfo</li>
  </ul>
</details>

afh reviewed (approved):
Thanks for updating this so quickly, @ddogfoodd, much appreciated.

Happy to merge this, once there is at least one approval from the package maintainers, just ping me :)

r-vdp commented:
I'll have a look tomorrow morning, I'm AFK for the weekend.

ddogfoodd commented:
drafting again because of a new follow-up bug fix release, waiting for graphics, then I will update both and undraft

ddogfoodd commented:
The bot also made a PR #454394
It seems to ignore graphics for now, do we close that?

ddogfoodd commented:
Agreed. I can close this PR then, right?

ddogfoodd commented:
> @ddogfoodd if you're motivated for it, you could rework this PR to split out the src derivations for the two packages, and then already bump alacritty to the new version, while keeping the fork still on the old one. If not, then I'll try to do so in the next days and push to this branch if you don't mind.

feel free to push, I don't feel like I know how to properly do it.

r-vdp committed:
alacritty: 0.15.1 -> 0.16.1

Diff: https://github.com/alacritty/alacritty/compare/v0.15.1...v0.16.1

Changelog: https://github.com/alacritty/alacritty/blob/v0.16.1/CHANGELOG.md

r-vdp committed:
alacritty-graphics: 0.15.0-graphics -> 0.16.0-graphics

Diff: https://github.com/ayosec/alacritty/compare/v0.15.0-graphics...v0.16.0-graphics

Changelog: https://github.com/alacritty/alacritty/blob/v0.16.0-graphics/CHANGELOG.md

r-vdp committed:
alacritty: make sure to put the right values for the homepage and changelog

r-vdp committed:
alacritty{-graphics}: make it possible to use different versions when the fork lags behind
